employees/tests/test_unit_report_viewset.py

```diff
@@ -327,6 +327,27 @@ def test_custom_report_list_create_serializer_method_should_return_serializer_wi
             serializer = view._create_serializer()
             self.assertEqual(serializer.fields["date"].initial, "2010-01-21")
 
+    def test_custom_report_list_create_serializer_method_should_return_serializer_filled_with_data_that_was_provided(
+        self
+    ):
+        from django.test import RequestFactory
```

Comment:
Please use `self.client` like we do in view tests, it should work here too and when we switch to Django views there will be less to change.

Reply:
![](https://i.ytimg.com/vi/js21k3WhT6M/hqdefault.jpg)

Reply:
Previously used solution works just as well, so import has been removed.